README_SFRAME_FORMAT.rst

```diff
@@ -1,15 +1,14 @@
-Blosc2 Extended-Frame Format
-============================
+Blosc2 Sparse Frame Format
+==========================
 
-Blosc (as of version 2.0.0) has an extended-frame format that allows for the storage of different Blosc data chunks
-sparse, on-disk.
+Blosc (as of version 2.0.0) has a sparse frame (sframe for short) format that allows for the storage of different Blosc data chunks on disk in a sparse way.
```

Comment:
Consider:

... allows for the _non-contiguous_ storage of ~~different~~ Blosc data chunks on disk.

Reply:
Yes, I like the (non-)contiguous qualifier.  Perhaps I'd change `storage->sequential` to `storage->contiguous`.  Agreed?

Reply:
I went ahead and did the change.  It looks better now :-)